index.html

```diff
@@ -216,10 +216,11 @@ <h2 class="h4">Miscellaneous</h2>
       <div data-hook="intro" class="columns intro center">
         <div class="row">
           <div class="new-property row text-left">
-            <p class="medium-14 columns">We're working on a <a href="https://property-beta.phila.gov/">new property search website</a> with more information and features.
+            <div class="medium-2 columns hide-for-small-only"><i class="fa fa-wrench fa-5x"></i></div>
+            <div class="medium-14 columns">We're working on a <a href="https://property-beta.phila.gov/">new property search website</a> with more information and features.
               We'd appreciate your feedback as we improve the new site.
-            </p>
-            <div class="medium-10 columns text-right">
+            </div>
```

Comment:
@karissademi idk if I'd really say that you "fixed" my markup here :) It seems like it was already semantically correct. Although I'll concede it looks about 100x better now.